packages/next/next-server/server/utils.ts

```diff
@@ -14,3 +15,28 @@ export function cleanAmpPath(pathname: string): string {
   pathname = pathname.replace(/\?$/, '')
   return pathname
 }
+
+export function collectEnv(page: string, pageEnv?: string[]): Env {
+  const missingEnvKeys = new Set()
+  const collected = pageEnv
+    ? pageEnv.reduce((prev: Env, key): Env => {
+        if (process.env[key]) {
+          prev[key] = process.env[key]!
+        } else {
+          missingEnvKeys.add(key)
+        }
+        return prev
+      }, {})
+    : {}
+
+  if (missingEnvKeys.size > 0) {
+    console.warn(
+      `Missing env value${missingEnvKeys.size === 1 ? '' : 's'}: ${[
+        ...missingEnvKeys,
+      ].join(', ')} for ${page}.\n` +
+        `Make sure to supply this value in either your .env file or in your environment.\n` +
+        `See here for more info: https://err.sh/zeit/next.js/missing-env-value`
```

Comment:
Shorthand: 
```suggestion
        `See here for more info: https://err.sh/next.js/missing-env-value`
```

Reply:
We use /zeit/next.js in 44 cases and /next.js in 20, think we should use zeit/next.js for everything